gitlabform/processors/abstract_processor.py

```diff
@@ -51,6 +51,16 @@ def process(
                 verbose(
                     f"Processing section '{self.configuration_name}' in dry-run mode."
                 )
+                try:
+                    project_transfer_source = configuration["project"]["transfer_from"]
+                    project_or_project_and_group = project_transfer_source
+                    verbose(
+                        f"""Project {project_or_project_and_group} is configured to be transferred, 
```

Comment:
The value of `project_or_project_and_group` is already modified to the same value as `project_transfer_source`. I think we need to save the original value for logging unless we can refer to it from another variable.

Reply:
Or... move the log one line above so that the change happens after the log.

Reply:
Moved the re-assignment below the verbose, is that ok?

Reply:
I think that should be fine. Thanks for the update.